assets/tests/models/routeTab.test.ts

```diff
@@ -16,3 +43,195 @@ describe("currentRouteTab", () => {
     expect(currentRouteTab([])).toMatchObject(routeTab)
   })
 })
+
+describe("isPreset", () => {
+  test("returns true for a saved preset", () => {
+    const routeTab = routeTabFactory.build({
+      presetName: "My Preset",
+      ordering: undefined,
+    })
+
+    expect(isPreset(routeTab)).toBeTruthy()
+  })
+
+  test("returns false for an open tab that is not a saved preset", () => {
+    const routeTab = routeTabFactory.build({
+      presetName: undefined,
+      ordering: 1,
+    })
+
+    expect(isPreset(routeTab)).toBeFalsy()
+  })
+})
+
+describe("isOpenTab", () => {
+  test("returns false for a saved preset that isn't open", () => {
+    const routeTab = routeTabFactory.build({
+      presetName: "My Preset",
+      ordering: undefined,
+    })
+
+    expect(isOpenTab(routeTab)).toBeFalsy()
+  })
+
+  test("returns true for an open tab that is not a saved preset", () => {
+    const routeTab = routeTabFactory.build({
+      presetName: undefined,
+      ordering: 1,
+    })
+
+    expect(isOpenTab(routeTab)).toBeTruthy()
+  })
+})
+
+describe("instantiatePresetByUUID", () => {
```

Comment:
**praise:** These tests are helpful for figuring out this function.

Reply:
Glad they helped! This and `closeTabByUUID` presented themselves as great opportunities for some TDD, so they actually helped me a lot in writing this function as well.